### PR TITLE
Bump kqoauth to qt5 build removing older versions

### PR DESCRIPTION
**Pull request by 0xd34df00d**
No revdeps except leechcraft, so should be safe.

### CODE REVIEW

**Comment by SoapGentoo** [comment]
@0xd34df00d can we also start nixing the leechcraft project? i.e., can you please go through all `metadata.xml`, and reassign leechcraft stuff to yourself + proxy-maint?

File: dev-libs/kqoauth/kqoauth-0.98-r2.ebuild [hunk 1]
```diff
@@ -1,31 +1,47 @@
-# Copyright 1999-2014 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
-# $Id$
+# $Header: /var/cvsroot/gentoo-x86/dev-libs/kqoauth/kqoauth-0.98-r1.ebuild,v 1.2 2014/07/11 17:54:39 zlogene Exp $
```
Comment (SoapGentoo) [inline dev-libs/kqoauth/kqoauth-0.98-r2.ebuild]: CVS header, bring back `# $Id$`

File: dev-libs/kqoauth/kqoauth-0.98-r2.ebuild [hunk 2]
```diff
@@ -1,31 +1,47 @@
...
 	sed -i -e '/SUBDIRS/s/ tests//' ${PN}.pro || die "sed on ${PN}.pro failed"
 	# respect libdir
 	sed -e 's:{INSTALL_PREFIX}/lib:[QT_INSTALL_LIBS]:g' -i src/src.pro || die "sed on src.pro failed"
 
-	qt4-r2_src_prepare
+	sed \
+		-e "s/TARGET = kqoauth/TARGET = kqoauth-qt5/g" \
+		-i src/src.pro || die
+
+	default
```
Comment (SoapGentoo) [inline dev-libs/kqoauth/kqoauth-0.98-r2.ebuild]: prefer to put the `default` (i.e. patching) as the first thing

File: dev-libs/kqoauth/kqoauth-0.98-r2.ebuild [hunk 3]
```diff
@@ -1,31 +1,47 @@
...
+}
+
+src_compile() {
+	eqmake5
```
Comment (SoapGentoo) [inline dev-libs/kqoauth/kqoauth-0.98-r2.ebuild]: shouldn't this happen in `src_configure`?

File: dev-libs/kqoauth/metadata.xml [hunk 4]
```diff
@@ -1,15 +1,8 @@
...
-    <email>leechcraft@gentoo.org</email>
-    <name>LeechCraft</name>
-  </maintainer>
-  <upstream>
-    <remote-id type="github">kypeli/kQOAuth</remote-id>
-  </upstream>
+	<maintainer type="person">
+		<email>0xd34df00d@gmail.com</email>
+		<name>Georg Rudoy</name>
+	</maintainer>
```
Comment (SoapGentoo) [inline dev-libs/kqoauth/metadata.xml]: don't forget proxy maint!

**Comment by 0xd34df00d** [comment]
Sure, I'll nix the leechcraft project. Expect that change with the update of lc-9999 ebuilds!

**Pull request closed without merging**